Extensions/TeamCity/Src/vss-extension.json

```diff
@@ -73,7 +73,7 @@
           },
           {
             "name": "Builds",
-            "endpointUrl": "{{endpoint.url}}httpAuth/app/rest/buildTypes/{{definition}}/builds",
+            "endpointUrl": "{{endpoint.url}}httpAuth/app/rest/buildTypes/{{definition}}/builds/?locator=branch:default:any",
```

Comment:
should we take the default branch to use for querying builds as input when linking the TeamCity artifact rather than listing build across all branches?